src/sentry/auth/idpmigration.py

```diff
@@ -15,8 +37,19 @@ def create_verification_key(user: User, org: Organization, email: str) -> None:
     :param org: the organization whose SSO provider is being used
     :param email: the email address associated with the SSO identity
     """
-    key = str(uuid.uuid4())  # noqa
-    raise NotImplementedError  # TODO
+    redis_key = "verificationKeyStorage"
+    TTL = 60 * 10
```

Comment:
These could both be top-level constants. I'd also recommend making them private-by-convention with a leading underscore (e.g., `_REDIS_KEY`).

Reply:
👍  I think i changed it to be a top level constant in the verification pr but didn't make it private, will make changes here too